lookup.js

```diff
@@ -21,16 +21,17 @@ const SSBURI = require('ssb-uri2')
 const validate = require('./validate')
 const FeedDetails = require('./FeedDetails')
 
-const SUBFEED_PREFIX_OFFSET = Math.max(
+const SUBFEED_PREFIX_OFFSET = (PARENTFEED_PREFIX_OFFSET = Math.max(
   '@'.length,
   'ssb:feed/bendybutt-v1/'.length,
   'ssb:feed/indexed-v1/'.length,
   'ssb:feed/gabbygrove-v1/'.length
-)
+))
 
 const B_VALUE = Buffer.from('value')
 const B_CONTENT = Buffer.from('content')
 const B_SUBFEED = Buffer.from('subfeed')
+const B_PARENT = Buffer.from('metafeed') // NOTE parent = metafeed in message content
```

Comment:
Let's name this `B_METAFEED`, because these `B_` buffers are a convention that arj and I have been using in db2 and other modules where we know that `B_FOO` is just a shortcut for `Buffer.from('foo')`. If you create an exception for this rule, then it's suddenly not anymore guaranteed that `B_FOO` stands for `Buffer.from('foo')` and can create confusion i.e. debugging nightmares.

Reply:
```suggestion
const B_METAFEED = Buffer.from('metafeed')
```